### PR TITLE
Walking skeleton

### PR DESCRIPTION
**Pull request by fifikim**
This PR meets all acceptance criteria including:
* detailed README
* Git hooks on pre-commit and pre-push
* Linting, unit testing, and addition of acceptance test suite
* CI workflow on push/pull request to all branches
* CD workflow on push to main branch
* Hello World demo that is executable from the production server via SSH connection

Demos:
1. Demo of SSH connection to the production server -- the root directory is empty before deployment: 
Documentation httpshelp ubuntu com

2. Demo of the deployment of a test file to the production server triggered by the CD workflow when code was pushed to the main branch: 
port 22

3. After code is pushed, the test file has been copied to the production server. (I added a rule to the repo after this push to prevent future pushes without a PR and approval) 
• Documentation httpshelp ubuntu com

** After this PR is merged to main, a .jar artifact will be deployed to the production server that prints "Hello World!" when executed.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+jobs:
+  build-and-test:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
```
Comment (ksdenisova) [inline .github/workflows/CI.yml]: This Java version looks very old. Can you change it to at least 17?

Comment (fifikim) [reply .github/workflows/CI.yml]: Yes! Will have to install 17 on my production server but I'll do that now and update the workflow when it's installed

File: .idea/workspace.xml [hunk 2]
```diff
@@ -5,19 +5,11 @@
   </component>
```
Comment (ksdenisova) [inline .idea/workspace.xml]: You possible need to add entire `.idea` directory  to .gitignore

File: .github/workflows/CI.yml (hunk 1 above)
Comment (fifikim) [reply .github/workflows/CI.yml]: installed & updated

**Review by ksdenisova — APPROVED** [review]
During the pair session, we agreed to add the acceptance tests to CI in the next `Simple Get` story. Ensure the CD job depends on CI and runs only after successfully passing CI.

Great job on documenting this PR and set up CD!

**Pull request merged**